tasks/main.yaml

```diff
@@ -0,0 +1,77 @@
+---
+- name: Install required pakages
+  package:
+    name: "{{ item }}"
+    state: present
+  with_items: "{{ capra_lvm_filesystem_packages }}"
+
+- name: "Configure default mountpoint directory"
+  file:
+    path: "{{ capra_lvm_filesystem_mount_parent }}"
+    state: directory
+    owner: "{{ capra_lvm_filesystem_owner }}"
+    group: "{{ capra_lvm_filesystem_group }}"
+    mode: 0755
+    seuser: unconfined_u
+    serole: object_r
+    setype: default_t
+    selevel: s0
+
+- name: "Unmount volume"
+  mount:
+    path: "{{ item.mount | default(capra_lvm_filesystem_mount_parent + '/' + item.name) }}"
+    state: absent
+  with_items: "{{ capra_lvm_filesystem_specs }}"
+  when: (item.state | default('present')) == 'absent'
+
+- name: "Create/remove logical volume"
+  lvol:
+    vg: "{{ item.vg | default(capra_lvm_filesystem_vg) }}"
+    lv: "{{ item.name | mandatory }}"
+    size: "{{ item.size | human_to_bytes }}B"
+    opts: "{{ capra_lvm_filesystem_lvcreate_options }}"
+    state: "{{ item.state | default('present') }}"
+    force: "{{ (item.state | default('present')) == 'absent' }}"
```

Comment:
I think it'd be better to split out the `present` and `absent` paths into separate, imported files. That way tags can also be used to selectively skip operations.

Reply:
I kept it all in one file as there are less than 100 lines but I splitter the paths and added tags.

Reply:
Responded on later revision.